apis/istio/v1alpha3/serviceentry_types.go

```diff
@@ -0,0 +1,390 @@
+package v1alpha3
+/*
```

Comment:
Honestly, nobody's copyright should be on these.  This entire folder was generated from protobuf with the types extracted.

Reply:
That is not true.  When I checked these files into `knative/serving` repository, which was later moved here, I didn't copy the generated code.

Reply:
Huh, I'm pretty confused then.

[This knative/..virtualservice_types.go#L35](https://github.com/knative/pkg/blob/master/apis/istio/v1alpha3/virtualservice_types.go#L35) sure looks like [this istio.../virtual_service.pb.go#L19](https://github.com/istio/api/blob/1.0.5/networking/v1alpha3/virtual_service.pb.go#L19) and [virtual_service.proto](https://github.com/istio/api/blob/1.0.5/networking/v1alpha3/virtual_service.proto#L58) from istio.  Surely you're not saying it's a coincidence?
I thought I was following your logic of just extracting the generated types - which the existing knative/istio types certainly are - but changing the spec so as to get deepcopy method.  They must match on the `json` and `proto` structure annotations or they simply won't work/match the istio crds.
My only point on this is that `istio` certainly should be on this copyright if anyone is...
Please clarify.

Reply:
No, it is certainly not an accident.  We manually translated the `.proto` files into `_types.go` file but  did not use the generate `*.pb.go` files.  Both files look like the `.proto` file, hence the similarity.  
 There are differences, for example like you observe that we didn't include the `proto` annotations since we don't use them.

However, like I said, since `knative/pkg` isn't designed to be vendored (except by other `knative/*` packages), if the missing `proto` annotations break your software you may have to fix it yourselves (I'm glad to help reviewing).

Regarding the copyright header we would like to keep it the same for all files in this repository.